Sources/MapboxMaps/Ornaments/Compass/MapboxCompassOrnamentView.swift

```diff
@@ -51,19 +51,25 @@ internal class MapboxCompassOrnamentView: UIButton {
                                               comment: "Accessibility hint")
 
         if let image = createCompassImage() {
-            containerView.image = image
-            NSLayoutConstraint.activate([
-                widthAnchor.constraint(equalToConstant: image.size.width),
-                heightAnchor.constraint(equalToConstant: image.size.height),
-                containerView.widthAnchor.constraint(equalToConstant: image.size.width),
-                containerView.heightAnchor.constraint(equalToConstant: image.size.height)
-            ])
+            updateImage(image: image)
```

Comment:
I believe [line 56](https://github.com/mapbox/mapbox-maps-ios/pull/1468/files#diff-b7e6600deb16d0a592c242ab3b13cdeb779b00639a5348dba82ab6452ef18903R56) should be placed before this `if` statement because it applies autolayout to the `containerView`

Reply:
interresting! Since this code was already this way :)
will patch